extensions/web3-profile/src/search-web3-profile.tsx

```diff
@@ -45,6 +45,7 @@ export default function Command() {
             <ActionPanel>
               <Action.Push
                 title="Show Profile"
+                icon={Icon.AppWindowSidebarLeft}
```

Comment:
nice touch 👌  think we may need to import `Icon` here

Reply:
Yes, good catch - I thought that I added it 😃

Reply:
running this locally, and i cant see any icon (`Icon.AppWindowSidebarLeft`) in the list 
<img width="750" alt="CleanShot 2022-09-05 at 20 37 08@2x" src="https://user-images.githubusercontent.com/372831/188501134-f5d7074d-8950-4433-86d7-5ca41102ec8e.png">

Reply:
That's strange, it should be a valid icon for quite some time now, does `npm install` do something?